dissect/target/plugin.py

```diff
@@ -837,7 +837,9 @@ def all_plugins():
         yield from os_plugins()
         yield from child_plugins()  # Doesn't export anything but added for completeness.
 
-    for available in all_plugins():
+    for available_original in all_plugins():
+        # Otherwise -f os.windows._os.users fails for instance
```

Comment:
```suggestion
        # Prevent modifying the global PLUGINS dict, otherwise -f os.windows._os.users fails for instance.
```

Gives maybe a bit more direction, on what this actually would do.